bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -79,6 +81,22 @@ trait UtilBijections {
   }
 
   /**
+   * Injection from java futures to twitter futures
+   */
+  implicit def twitter2JavaFuture[A](
+    implicit converter: JavaFutureToTwitterFutureConverter
+  ): Bijection[TwitterFuture[A], JavaFuture[A]] = {
+    new AbstractBijection[TwitterFuture[A], JavaFuture[A]] {
+      override def apply(f: TwitterFuture[A]): JavaFuture[A] =
+        f.toJavaFuture.asInstanceOf[JavaFuture[A]]
```

Comment:
Since I can't use `CompletableFuture` I found this way around, I don't know if there is a better way, feedback welcome.